bin/dev_host_bridge.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Write the network of the host to the /etc/hosts file if "host.docker.internal"
+# is not supported on the system.
+# From: https://dev.to/bufferings/access-host-from-a-docker-container-4099
+
+# Check if we are on a supported system
+HOST_DOMAIN="host.docker.internal"
+ping -q -c1 $HOST_DOMAIN > /dev/null 2>&1
+
+# If not, then write find our IP and map "host.docker.internal" to it
+if [ $? -ne 0 ]; then
+  HOST_IP=$(ip route | awk 'NR==1 {print $3}')
+  echo -e "$HOST_IP\t$HOST_DOMAIN" >> /etc/hosts
+fi
```

Comment:
This is lifted from the same solution in ViaHTML to support Mac and linux folk. For a refreshing and annoying change, Linux is the odd one our here which doesn't support "host.docker.internal" by default (Mac and Windows do).

Why? Who knows.

Reply:
This writes to the `/etc/hosts` file in the Docker container, right? Not on my host system

Reply:
Yes. This is executed inside the docker container as the first part of the command run. See the `docker-compose.yml` for more details.